install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [[ $(whoami) != "root" ]]; then
+  echo "This file needs to be executed as root or with sudo."
+  exit
+fi
+
+defaultdir="/usr/bin"
+
+echo "Directory to install to (default: $defaultdir)"
+read installdir
+
+if [[ $installdir = "" ]]; then
+  installdir=$defaultdir
+fi
+
+if [[ ! -d $installdir ]]; then
+  echo "This directory does not exist."
+  exit
+fi
+
+cd src
+zip -r ../pridefetch.zip *
+cd ..
+echo "#!/usr/bin/env python" | cat - pridefetch.zip > pridefetch
+cp pridefetch "$installdir/"
+chmod 755 "$installdir/pridefetch"
+
+if [[ -f "$installdir/pridefetch" ]]; then
+  echo -e "\nPridefetch installed successfully"
+  exit
+else
+  echo -e "\nSomething went wrong"
```

Comment:
`Something went wrong, and pridefetch couldn't install properly.`

Would sound better here